src/pages/EnablePayments/IdologyQuestions.js

```diff
@@ -131,7 +128,17 @@ function IdologyQuestions({questions, walletAdditionalDetails, idNumber}) {
                     {translate('additionalDetailsStep.helpLink')}
                 </TextLink>
             </View>
-            <FormScrollView ref={formRef}>
+            <FormProvider
+                customErrorMessage={errorMessage}
+                submitButtonStyles={[styles.mh5, styles.mb5, styles.justifyContentEnd]}
```

Comment:
1. Remove `submitButtonStyles` prop
2. Change style to `{[styles.flexGrow1, styles.ph5]}`
3. Removing the wrapping view inside

It should look something like the following (assuming the rest of the comments are solved):

```js
            <FormProvider
                formID="walletAdditionalDetails"
                validate={validate}
                onSubmit={submitAnswers}
                scrollContextEnabled
                style={[styles.flexGrow1, styles.ph5]}
                submitButtonText={translate('common.saveAndContinue')}
            >
                <InputWrapper
                    InputComponent={SingleChoiceQuestion}
                    inputID="answer"
                    prompt={currentQuestion.prompt}
                    possibleAnswers={possibleAnswers}
                    currentQuestionIndex={currentQuestionIndex}
                    onValueChange={chooseAnswer}
                />
            </FormProvider>

```